src/test/unit/__snapshots__/profile-conversion.test.js.snap

```diff
@@ -573,7 +573,7 @@ Object {
       "name": "Chrome Thread",
       "pausedRanges": Array [],
       "pid": 88999,
-      "processName": "Renderer",
+      "processName": "Renderer (Wandering Lines #2)",
```

Comment:
It makes me somewhat happy to find my art show up in a project. I'd forgotten I'd profiled that.